game/scripts/npc/abilities/storm_spirit_static_remnant.txt

```diff
@@ -52,7 +52,7 @@
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "static_remnant_damage"                           "120 175 230 285 340 395"
+        "static_remnant_damage"                           "120 175 230 285 400 600"
```

Comment:
why though? this goes against the point of the patch...

Reply:
Because the scaling is bad? Also I asked Naeil about this, he told me to buff every level of remnant, lmao.